src/streaming/MediaManager.js

```diff
@@ -114,7 +114,7 @@ function MediaManager() {
 
             // Create a prebuffered player
             const prebufferedPlayer = MediaPlayer().create();
-            prebufferedPlayer.initialize(null, alternativeMpdUrl, false, NaN, alternativeContext);
+            prebufferedPlayer.initialize(null, alternativeMpdUrl, false, NaN, null);
```

Comment:
```suggestion
            prebufferedPlayer.initialize(null, alternativeMpdUrl, false, NaN);
```

I think the alternativeContext is not needed anymore

Reply:
If not you should remove it in the MediaPlayer.initialize method